openapi/spec-generator/main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"fmt"
+	"os"
+)
+
+func main() {
```

Comment:
I'd suggest accepting the output path here and parsing the args.

Also, it will be nice to add a usage/`--help` output. I tried running the command, which exitted cleanly but without it being clear to me that `../v1/openapi.yaml` was being updated.